services/web/server/src/simcore_service_webserver/notifications/_rabbitmq_consumers.py

```diff
@@ -169,7 +170,7 @@ async def _events_message_parser(app: web.Application, data: bytes) -> bool:
     (
         LoggerRabbitMessage.get_channel_name(),
         _log_message_parser,
-        {},
+        dict(topics=["#"]),
```

Comment:
MINOR: perhaps you can add a constant to document what ``#`` means?

Reply:
it is temporary. I will remove it in the later PRs

Reply:
but you are of course right. will do